src/main/java/de/rabea/game/Game.java

```diff
@@ -1,64 +1,54 @@
 package de.rabea.game;
 
-import de.rabea.player.GuiPlayer;
-import de.rabea.player.UnbeatableComputerPlayer;
-
 public class Game {
 
     private final UserInterface userInterface;
-    private final Player player;
-    private final Player opponent ;
+    private final Player player1;
+    private final Player player2;
     private Player currentPlayer;
+    private Player notCurrentPlayer;
 
-    public Game(UserInterface userInterface, Player player, Player opponent) {
+    public Game(UserInterface userInterface, Player player1, Player player2) {
         this.userInterface = userInterface;
-        this.player = player;
-        this.opponent = opponent;
-        this.currentPlayer = player;
+        this.player1 = player1;
+        this.player2 = player2;
+        this.currentPlayer = player1;
+        this.notCurrentPlayer = player2;
```

Comment:
Having the 'notCurrentPlayer' feels odd... Is this to solve an off-by-one issue with switching players?